### PR TITLE
Creates a field for tracking negative member reaction responses. Creates field for assigning an Integer value to a reaction.

### PR DESCRIPTION
**Pull request by eet349**
Description
Creates a field for tracking negative member reaction responses. Creates field for assigning an Integer value to a reaction.
Why was this work done?  
We needed a better way to store the value information for the member reactions. These values are going to be used for analysis to display data visualizations in the dashboard.
We also are going to work on a alerts feature so we added a field for keeping track if those alerts have been handled or not.

[x] New feature (non-breaking change which adds functionality)
Change Status  
[x] Completed, ready to review and merge  
Has This Been Tested  
[x] Yes
Checklist
[x] My code follows the style guidelines of this project  
[x] I have performed a self-review of my own code  
[x] My code has been reviewed by at least one peer  
[x] I have commented my code, particularly in hard-to-understand areas  
[] I have made corresponding changes to the documentation  
[x] My changes generate no new warnings  
[x] There are no merge conflicts

### CODE REVIEW

File: src/main/java/com/lambdaschool/oktafoundation/SeedData.java [hunk 1]
```diff
@@ -227,34 +227,43 @@ public void run(String[] args) throws
...
+                "1F610",    //  3
+                "1F641",    //  2
+                "1F61E",    //  1
+                // Checkin/checkout emojis
+                "1F60A",    //  5
+                "1F60E",    //  4
+                "1F974",    //  3
+                "1F634",    //  2
+                "1F62D",    //  1
+                //  = Currently unused emojis
```
Comment (L-Steinmacher) [inline src/main/java/com/lambdaschool/oktafoundation/SeedData.java:244]: Nice comments on the unused emojis.

File: src/main/java/com/lambdaschool/oktafoundation/models/MemberReactions.java [hunk 2]
```diff
@@ -25,6 +26,12 @@
     @GeneratedValue(strategy =  GenerationType.AUTO)
     private long memberreactionid;
 
+    /**
+     * The  reactionresolved(boolean) for memberreactions lets us know if there has been a negative emotion entered as a memeberreaction.
+     */
+    @Column(nullable = false)
+    private boolean reactionresolved;
+
```
Comment (L-Steinmacher) [inline src/main/java/com/lambdaschool/oktafoundation/models/MemberReactions.java:34]: Still more excellent commenting.  Straight forward boolean

File: src/main/java/com/lambdaschool/oktafoundation/models/Reaction.java [hunk 3]
```diff
@@ -25,12 +25,26 @@
     @Column(nullable = false, unique = true)
     private String reactionvalue;
 
+    /**
+     * The reactionint (Integer). Cannot be null and must be unique
+     * Will hold the value for the reaction to be used for analysis.
+     */
+    @Column(nullable = false, unique = false)
+    private Integer reactionint;
+
```
Comment (L-Steinmacher) [inline src/main/java/com/lambdaschool/oktafoundation/models/Reaction.java:34]: Straight forward constructor and model.  Appreciate the comments once again!

**Review by L-Steinmacher — APPROVED** [review]
Rather straight forward  model design.  The removal of the unused emojis as well as keeping them there for future use  with comments is great!  Clean code. Looks great!

**Review by JackHenry15 — APPROVED** [review]
Pair programmed with you and Kay for a lot of this. Annotated well and looks good to go! Merging this.

**Pull request merged**